Portal/src/Datahub.Portal/Pages/Workspace/Toolbox/WorkspaceToolboxPage.razor

```diff
@@ -120,7 +123,9 @@
     {
         Available,
         SendingRequest,
+        SendingDelete,
         PendingRequest,
+        PendingDeleteRequest,
```

Comment:
Are there UI updates for these statuses? I can see the updates but no corresponding blazor

Reply:
Updated razor code as well